src/widgetastic/widget/__init__.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+"""This module contains the base classes that are used to implement the more specific behaviour."""
+from .base import *  # noqa: F403 F401
+from .checkbox import Checkbox  # noqa: F401
+from .image import Image  # noqa: F401
+from .input import BaseInput, ColourInput, FileInput, TextInput  # noqa: F401
+from .select import Select  # noqa: F401
+from .table import Table, TableColumn, TableRow  # noqa: F401
+from .text import Text  # noqa: F401
+
+# for backward compatibility, it is incorrectly used in integration_tests repo
+from widgetastic.exceptions import RowNotFound  # noqa: F401
```

Comment:
Let's fix that ... :)

https://github.com/ManageIQ/integration_tests/pull/7919/files